Solita.LocalizationEditor.UI/DAL/VppFileAccessStrategy.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Web;
+using System.Web.Hosting;
+using System.Xml;
+using EPiServer.Web.Hosting;
+using Solita.LocalizationEditor.UI.Common;
+using Solita.LocalizationEditor.UI.Models;
+
+namespace Solita.LocalizationEditor.UI.DAL
+{
+    //public class VppFileAccessStrategy : FileAccessStrategy
```

Comment:
I assume this is commented out because it's not compatible with EPi 9? Should it be moved to a separate library?